pylintrc

```diff
@@ -364,6 +364,7 @@ disable =
 	no-else-continue,
 	useless-object-inheritance,
     useless-suppression,
+    cyclic-import,
```

Comment:
@UsamaSadiq  These changes should not be made to `pylintrc` directly, but added to `pylint_tweaks` following the guidlines at the top of `pylintrc`. Future updates to rule changes will overwrite these if they aren't added there.